concepts/conditionals/about.md

```diff
@@ -96,9 +96,11 @@ end
 # => "1 is equal to 1"
 
 unless value < 2
-  "1 is not greater than 2"
+  "1 is not less than 2"
+else
+  "1 is less than 2"
```

Comment:
Because there is an `else` here it strongly suggests that this should be an `if` statement instead, where we swap the conditional statements to be correct.  Avoiding `else` with `unless` is very idiomatic, as it prevents mental gymnastics that otherwise happens.